### ISSUE TITLE
[Accessibility] Support task completion/failure audio cues in Output

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#18955]
CC -meganrogge 

Many extensions, such as [microsoft/vscode-arduino](https://github-com/microsoft/vscode-arduino), use Output view and display compile results like below:

![image](https://user-images.githubusercontent.com/19754711/223737392-4a0cfa2f-1c44-4c3e-b24b-5c54b6b87cd7.png)

Could you please support task completion and failure audio cues in output view as well? Currently, it is very tedious to read through the results line by line.
